packages/strapi-plugin-upload/config/schema.graphql.js

```diff
@@ -88,7 +88,7 @@ const formatFile = async (upload, metas) => {
       type: mimetype,
       size: buffer.length,
     },
-    {},
+    extraInfo,
```

Comment:
in case we send a null value it will fail make sure to default to en empty object here :)

Reply:
Then how about `extraInfo || {}`?

Reply:
Exactly :)